base/src/Data/Macaw/Memory.hs

```diff
@@ -421,10 +421,10 @@ memInt =
     Addr64 -> \x -> MemInt x
 
 instance Eq (MemInt w) where
-  (==) = \x y -> memIntValue x == memIntValue y
+  (==) x y = memIntValue x == memIntValue y
```

Comment:
A very minor thing, but if we're going to bother eta-expanding a function that's meant to be used infix, we might as well define it infix as well:

```suggestion
  x == y = memIntValue x == memIntValue y
```